### PR TITLE
Add more methods to checkout client

### PR DESCRIPTION
**Pull request by gsasouza**
#### What is the purpose of this pull request?

Add more methods to checkout client base on [store-graphql checkout client](https://github.com/vtex-apps/store-graphql/blob/master/node/clients/checkout.ts)
#### What problem is this solving?

lack of some method in the default checkout client
#### Types of changes
- [ ] Refactor (non-breaking change that only makes the code better)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Chores checklist
- [ ] Update `CHANGELOG.md`

### CODE REVIEW

**Comment by lucis** [comment]
That's an amazing work, @gsasouza!

One thing I want to hear from you is the choosing of `AUTH_TOKEN` as the auth strategy. This would probably "work" but would mean that the request is going to be made **on behalf of the app**, and not of the user like [`store-graphql` does](https://github.com/vtex-apps/store-graphql/blob/master/node/clients/checkout.ts#L16).

I think you'd probably want to use `STORE_TOKEN` on pretty much all methods, since these operations should happen on "user land". Using the app's token could, in some scenarios, expose a security vulnerability.

@FabianaFerreira may help here. The current method we have on `Checkout` client was really supposed to be `AUTH_TOKEN`? I don't know what `order form configuration' represents.

Again, great work 🚀  Let me know when you've finished the PR so we can release it right-away

**Comment by gsasouza** [comment]
Hi @lucis,

To be honest, I don't know much about the auth methods and I just tried to follow the same pattern that was in the previous functions. I'll change the new methods to `STORE_TOKEN` since is the way store-graphql does.

**Comment by fabianaferreira** [comment]
I'll make Lucis' words mine, it's an amazing work what you did here. Thank you, @gsasouza! 🚀 

> @FabianaFerreira may help here. The current method we have on `Checkout` client was really supposed to be `AUTH_TOKEN`? I don't know what `order form configuration' represents.

The Order Form configuration is the object that stores a lot of settings on the account, such as gift card providers, tax providers, and so on. With that in mind, considering that it's not something that we usually do on "userland", I believe it's okay to have it with the app token. What do you think, @lucis? It's used in cases of integration for gift card and tax providers. I do not have thaaat context on this matter, but I know these use cases because I've worked with them before.

File: src/clients/checkout.ts [hunk 1]
```diff
@@ -29,24 +65,33 @@ export class Checkout extends JanusClient {
     })
   }
 
+  private getRequestConfig(
```
Comment (fabianaferreira) [inline src/clients/checkout.ts]: what do you think about making this method a util function? in that case, we could refactor the other clients' code to use it... passing also the context as a parameter. what do you think about this, @lucis?

Comment (gsasouza) [reply src/clients/checkout.ts]: Good idea!

**Review by fabianaferreira — APPROVED** [review]
Niiiiice 🚀

**Pull request merged**